week-1/implement/max.js

```diff
@@ -0,0 +1,23 @@
+function maxNumber(num) {
+    if (num.length === 0) {
+        return -Infinity; 
+    }
+
+    if (num.length === 1 && typeof num === 'number') {
```

Comment:
What are you trying to do by writing this condition?

Reply:
It is for checking if the input num is an array with exactly one element, and if that element is a number. If this condition is true, the function returns that single number.